### ISSUE TITLE
Implement tests for the transition domain file parse and loading

### ISSUE THREAD

**Issue by abroddrick** [cisagov/manage.get.gov#1126]
### Issue Description

The implementation for  #923 (parse text files and load data into the TransitionDomain table) and #926  didn't include test cases and it was decided to break out the test case portion into it's own ticket due to the complexity and so other tickets could be unblocked. The current implementation does seem to work but it should be expected that something may have to be fixed on the current implementation as a result of testing

AC:
- create unit tests for #923 ([see PR ](https://github.com/cisagov/getgov/pull/1077)
- create unit tests for #926, --->seePR  #1115 
- fix any bugs that are found

out of scope:
no need to test debug prints or colors

**Comment by PaulKuykendall** [comment]
this should all have been covered in other ways/tickets.
